code/controllers/subsystem/lobotomy_corp.dm

```diff
@@ -236,6 +236,8 @@ SUBSYSTEM_DEF(lobotomy_corp)
 			pod.explosionSize = list(0,0,0,0)
 			to_chat(person, "<span class='nicegreen'>It's pizza time!</span>")
 			new /obj/effect/pod_landingzone(get_turf(person), pod)
+
+			GLOB.realization_charges +=1
```

Comment:
wont putting this here make dead agents/clerks also give additional charges to the realization engine?
eighter that or the comment at the top is wrong

Reply:
Oh wait yeah, it does do that.
I actually got distracted mid-making this PR and hotswapped to doing something cooler.